hfc/fabric/client.py

```diff
@@ -1597,6 +1598,12 @@ def cc_event(self, cc_event, block_number, tx_id, tx_status):
         :param wait_for_event_timeout: Time to wait for the event from each
          peer's deliver filtered service signifying that the 'invoke'
           transaction has been committed successfully (default 30s)
+        :param grpc_broker_unavailable_retry: Number of retry if a brocker
+         is unavailable (default 0)
+        :param grpc_broker_unavailable_retry_delay : Delay in ms to retry
+         (default 3000 ms)
+        :param raise_broker_unavailable: Raise if any brocker is unavailable,
+         else send the proposal
```

Comment:
potential comment improvement:

```suggestion
         else always send the proposal regardless of unavailable brokers.
```